ccmlib/scylla_repository.py

```diff
@@ -249,9 +246,9 @@ def run_scylla_install_script(install_dir, target_dir, package_version):
         # on relocatable package format 2.1 or later, use --packaging instead of sed
         install_opt = ' --packaging'
 
-    run('''{0}/install.sh --root {1} --prefix {1} --prefix /opt/scylladb --nonroot{2}'''.format(
+    run('''{0}/install.sh --prefix {1} --nonroot{2}'''.format(
```

Comment:
You may want to add --packaging (maybe in a separate patch) to avoid installation of a scylla-server.service under the user's systemd configuration.

Reply:
It's added based on the package version.
See https://github.com/scylladb/scylla-ccm/blob/74547ec5a3b92201fd6569c388eadf762d8c58a5/ccmlib/scylla_repository.py#L244-L253